bedrock/base/templates/includes/protocol/footer/footer-refresh.html

```diff
@@ -17,16 +17,13 @@ <h2 class="moz24-footer-label" data-testid="footer-heading-company">
               {{ ftl('footer-refresh-company') }}
             </h2>
             <ul class="moz24-footer-primary-list" data-testid="footer-list-company">
-              <li><a href="{{ url('mozorg.about.leadership.index') }}" data-link-position="footer" data-link-text="Leadership">{{ ftl('footer-refresh-leadership') }}</a></li>
+              <li><a href="https://www.mozilla.org/{{ LANG }}/about/leadership/?{{ utm_params }}" data-link-position="footer" data-link-text="Leadership">{{ ftl('footer-refresh-leadership') }}</a></li>
```

Comment:
Question: where do `utm_params` come from for this template? In the non-refresh footer they appear to be set in the template.


Suggestion:
Instead of `?{{ utm_params }}`, which will always leave a trailing `?` even if there's no `utm_params` set, could we wrap it in a helper that takes context so we can conditionally include it? Something like:

```
@library.global_function
@jinja2.pass_context
def add_utm_params(ctx):
     if 'utm_params' in ctx:
         return f"?{ctx['utm_params']}"
     return ""
```

used like this maybe?

```
<a href="https://www.mozilla.org/{{ LANG }}/about/leadership/{{ set_utm_params() }}"
```

Reply:
It’s already set in the base template for the footer, so applies to both versions